app/core/serializers.py

```diff
@@ -0,0 +1,43 @@
+from rest_framework.serializers import ModelSerializer, Serializer
+
+from core.models import User, Event
+
+
+class HomeSerializer(Serializer):
+    class Meta:
+        model = Event
+        fields = (
+            'name',
+            'organizer',
+            'tags',
+            'date',
+        )
+
+
+class EventSerializer(ModelSerializer):
```

Comment:
consider updating date to current date while update()

Reply:
obsolete